app/abr-menu.js

```diff
@@ -30,6 +31,10 @@ function preprocessTemplate (element, config) {
             win.webContents.send("command", command, parameters);
         },
         replaceAttributes = function (item) {
+            if (item.label) {
+                item.label = localize.get(config, item.label);
```

Comment:
> My guess is that using the item ID would prevent sharing a key between items. Am I right?

Yes, you are. It's definitely better to use a dedicated key as you said.

> Maybe we should create a new property "key" to items in the JSON files. When localizing an item, we could:
> * Check if it has a "key" property but misses a "label" property
> * If so, add the "label" property set with the translation and remove the "key" property

I would try the opposite:

* If `key` exists, then replace `label` with `localize(key)` and `delete key`
* Otherwise keep `label` as is

This allows to keep current labels as default values to ensure that a label will always be provided (which is especially important if we decide to move translations files to user config directory).